src/translations/en.json

```diff
@@ -1,4 +1,17 @@
 {
+  "card": {
+    "camera": {
+      "not_available": "Image not available"
+    },
+    "persistent_notification": {
+      "dismiss": "DISMISS"
```

Comment:
Just as a nitpick, this should just be `"dismiss": "Dismiss"`. The CSS for the button will make it all caps. We try to keep the stored translations in sentence case, and use CSS if we need to change it.